credentials/src/main/kotlin/web5/sdk/credentials/model/PresentationSubmission.kt

```diff
@@ -0,0 +1,29 @@
+package web5.sdk.credentials.model
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.annotation.JsonProperty
+
+/**
+ * Represents a presentation submission object.
+ *
+ * @see [Presentation Submission](https://identity.foundation/presentation-exchange/spec/v2.0.0/#presentation-submission)
+ */
+public data class PresentationSubmission(
+  val id: String,
+  @JsonProperty("definition_id")
+  val definitionId: String,
+  @JsonProperty("descriptor_map")
+  val descriptorMap: List<DescriptorMap>
+)
+
+/**
+ * Represents descriptor map for a presentation submission.
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public data class DescriptorMap(
```

Comment:
For binary compatibility when updating files. 
```suggestion
public class DescriptorMap(
```